glean-core/src/upload/directory.rs

```diff
@@ -86,13 +94,16 @@ impl PingDirectoryManager {
     ///
     /// * `uuid` - The UUID of the ping file to be processed
     pub fn process_file(&self, uuid: &str) -> Option<PingRequest> {
-        let path = self.get_dir().join(uuid);
-        let file = match File::open(path) {
+        let path = match self.get_file_path(uuid) {
+            Some(path) => path,
+            _ => return None,
```

Comment:
Do we expect this to happen? If yes, can you document why? If not, can you at least log an error?

Reply:
No we dont expect it. I will log an error.